Docs/source/building/cmake.rst

```diff
@@ -131,7 +131,7 @@ CMake Option                  Default & Values                             Descr
 ``WarpX_PARSER_DEPTH``        **24**                                       Maximum parser depth for input file functions
```

Comment:
Above in line `34` the mention of `Boost 1.66.0+: for QED support` can be updated, too :)
```
Boost 1.66.0+: Support for QED tables generation
```

Reply:
Also the required dependencies in need a new line 27:
```
- `PICSAR <https://github.com/ECP-WarpX/picsar>`_: we automatically download and compile a copy of PICSAR
```

Reply:
I should have updated the documentation...